### DISCUSSION TITLE
[功能改进建议]: 刷流插件设置启用时间段

### DISCUSSION THREAD

**Discussion by jackloves111** [jxxghp/MoviePilot#3414] [讨论]
### 当前程序版本

1.3.6

### 运行环境

Docker

### 功能改进类型

插件

## 目前的插件设置，没有启用时间段的设置，只有启用或不启用，我的设想是在夜间闲时执行，白天停止，以免影响宽带和性能。

![image](https://github.com/jxxghp/MoviePilot/assets/89971817/340288db-f7e9-4fc4-9a0b-ff0823238b5c)

## 原nt是可以用cron进行设置的

**Comment by Mikcoono1** [comment]
为啥我点配置 不弹你这个 页面出来.... 都升级到 1.3.9-2了
